source/manual/local-frontend-development.html.md

```diff
@@ -0,0 +1,84 @@
+---
+title: Local frontend development
+parent: "/manual.html"
+layout: manual_layout
+section: Frontend
+type: learn
+owner_slack: "#govuk-2ndline"
+last_reviewed_on: 2019-07-08
+review_in: 6 months
+---
+
+Make the following changes to your setup if you are making changes to frontend apps and need to view those changes locally.
+
+In the instructions below we are running the [frontend] app, but would work for any of the other frontend apps such as [government-frontend].
```

Comment:
```suggestion
In the instructions below we are running the [frontend] app, but this would work for any of the other frontend apps such as [government-frontend].
```